### PR TITLE
Gives Cinq a hat

### PR DESCRIPTION
**Pull request by KingSuperBOB**
## About The Pull Request
Implements the Cinq Hat sprite that Jim Lobotomite made, the sprite is very good, please praise them.
image

## Why It's Good For The Game
The admins have likely grown tired of spawning in the Bounty Hunter hat and painting it blue for Cinq fixers

## Changelog
:cl:
add: Implements the Cinq Hat sprite made by Jim lobotomite
/:cl:

### CODE REVIEW

File: code/modules/clothing/suits/ego_gear/non_abnormality/cinq.dm [hunk 1]
```diff
@@ -3,3 +3,9 @@
 	desc = "A black and blue suit used by the fixers of cinq association."
 	icon_state = "cinq"
 	armor = list(RED_DAMAGE = 30, WHITE_DAMAGE = 30, BLACK_DAMAGE = 30, PALE_DAMAGE = 0)
+	hat = /obj/item/clothing/head/ego_hat/cinq
+
+/obj/item/clothing/head/ego_hat/cinq
+	name = "cinq association cavalier hat"
+	desc = "A cavalier hat worn by members of the qinq qssociation, it gives you the urge to d-d-d-duel"
```
Comment (Kitsunemitsu) [inline code/modules/clothing/suits/ego_gear/non_abnormality/cinq.dm:10]: Minor spelling mistake.

File: code/modules/clothing/suits/ego_gear/non_abnormality/cinq.dm [hunk 2]
```diff
@@ -3,3 +3,9 @@
 	desc = "A black and blue suit used by the fixers of cinq association."
 	icon_state = "cinq"
 	armor = list(RED_DAMAGE = 30, WHITE_DAMAGE = 30, BLACK_DAMAGE = 30, PALE_DAMAGE = 0)
+	hat = /obj/item/clothing/head/ego_hat/cinq
+
+/obj/item/clothing/head/ego_hat/cinq
+	name = "cinq association cavalier hat"
+	desc = "A cavalier hat worn by members of the cinq qssociation, it gives you the urge to d-d-d-duel"
```
Comment (Kitsunemitsu) [inline code/modules/clothing/suits/ego_gear/non_abnormality/cinq.dm:10]: Minor spelling mistake

**Review by Kitsunemitsu — APPROVED** [review]
Minor spelling correction

**Comment by KingSuperBOB** [comment]
[imagem_2024-05-17_020830285](https://github.com/vlggms/lobotomy-corp13/assets/88884857/d0abdca5-bcff-426b-bca1-f87d35435a5d)

**Pull request merged**